.travis.yml

```diff
@@ -1,10 +1,14 @@
 language: php
 
-php:
-- 7.0
-- 7.1
-- 7.2
-- 7.3
+matrix:
+  fast_finish: true
+  include:
+    - php: 7.0
+    - php: 7.1
+    - php: 7.2
+    - php: 7.3
+    - php: 7.4
+      env: FULL_TEST=1
```

Comment:
The error given in the last failing build:
https://travis-ci.org/github/paragonie/paseto/jobs/677231516#L318-L321
is due to Psalm bugs fixed in more recent version, but not in the older ones.
In general, test shall be done on every version, of course, but static analysis is enough only on the latest one.